test/agent/README.md

```diff
@@ -29,6 +29,14 @@ One way to run the traffic test is as follows
 #### Networking Testing
 Networking testing binary must be invoked on a pod that runs in Host Networking mode. It is capable of testing the Pod networking is setup correctly and once the Pod has been deleted the networking has been teared down correctly by the CNI Plugin.
 
+### Using the local changes to Agent inside ginkgo 
+Ginkgo tests take dependency on the Agent module for supplying PodValidationInput as of now.
+If you are making changes to the PodValidationValidationInput then gingko must use local copy of agent with your modifications instead of downloading the upstream module to reflect those changes.
```

Comment:
Somehow my previous comment got missed. This looks like a typo? `PodValidationValidationInput `

Reply:
fixed